Index: src/Figures/ChessFigure.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/Figures/ChessFigure.java b/src/Figures/ChessFigure.java
--- a/src/Figures/ChessFigure.java	
+++ b/src/Figures/ChessFigure.java	
@@ -1,7 +1,7 @@
 package Figures;
 
 public abstract class ChessFigure {
-    private boolean color;
+    private boolean color; /* TRUE = NEGRAS ; FALSE = BLANCAS */
     protected String nameFigure;
 
     public ChessFigure (String nameFigure, boolean color) {
Index: src/Figures/Queen.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/Figures/Queen.java b/src/Figures/Queen.java
--- a/src/Figures/Queen.java	
+++ b/src/Figures/Queen.java	
@@ -1,5 +1,7 @@
 package Figures;
 
+import Mecanicas.*;
+
 public class Queen extends ChessFigure {
     public Queen(boolean color) {
         super(color);
@@ -10,7 +12,10 @@
     }
 
     @Override
-    public void movement() {
-        //Move
+    public boolean movement(Movement mov, Tablero tb) {
+        return mov.esDiagonal() || mov.esHorizontal() || mov.esVertical();
+
     }
 }
+
+
