Index: src/Figures/horse.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package Figures;\r\n\r\npublic class horse {\r\n}\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/Figures/horse.java b/src/Figures/horse.java
--- a/src/Figures/horse.java	(revision 243142ed68987795b49b03ab6767d78f8a5de87c)
+++ b/src/Figures/horse.java	(date 1674636700656)
@@ -1,4 +1,16 @@
 package Figures;
 
-public class horse {
+public class Horse extends ChessFigure {
+    public Horse(boolean color) {
+        super(color);
+        if (color)
+            nameFigure = "[♞]";//negra
+        else
+            nameFigure = "[♘]";//blanca
+    }
+
+    @Override
+    public void movement() {
+        //Hace movimiento
+    }
 }
