Index: src/Figures/horse.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package Figures;\r\n\r\npublic class Horse extends ChessFigure {\r\n    public Horse(boolean color) {\r\n        super( color);\r\n        if (color)\r\n            nameFigure = \"[♞]\";//negra\r\n        else\r\n            nameFigure = \"[♘]\";\r\n    }\r\n\r\n    @Override\r\n    public void movement() {\r\n        //Move\r\n    }\r\n\r\n}\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/Figures/horse.java b/src/Figures/horse.java
--- a/src/Figures/horse.java	(revision 117e88449587f3e3890a26e76cfee966e7b29ae7)
+++ b/src/Figures/horse.java	(date 1674638019713)
@@ -2,16 +2,15 @@
 
 public class Horse extends ChessFigure {
     public Horse(boolean color) {
-        super( color);
+        super(color);
         if (color)
             nameFigure = "[♞]";//negra
         else
-            nameFigure = "[♘]";
+            nameFigure = "[♘]";//blanca
     }
 
     @Override
     public void movement() {
-        //Move
+        //Hace movimiento
     }
-
 }
